torchx/lib/torchx/backend.ex

```diff
@@ -642,6 +642,13 @@ defmodule Torchx.Backend do
     |> to_nx(out)
   end
 
+  @impl true
+  def eigh({eigenvals, eigenvecs}, tensor, _opts) do
+    {q, r} = Torchx.eigh(from_nx(tensor))
```

Comment:
```suggestion
  def eigh({eigenvals, %T{type: type} = eigenvecs}, tensor, _opts) do
    {q, r} =
      tensor
      |> from_nx()
      |> Torchx.to_type(to_torchx_type(type))
      |> Torchx.eigh()
```

Reply:
it should be {eigenvecs, eigenvals} instead of {q, r}

Reply:
And I would rename the output holder by adding the _out suffix